src/utils/actionUtils.ts

```diff
@@ -60,8 +60,9 @@ export function getInputAsArray(
     return core
         .getInput(name, options)
         .split("\n")
-        .map(s => s.trim())
-        .filter(x => x !== "");
+        .map(s => s.replace(/^\!\s+/, '!').trim())
```

Comment:
Can you explain what does this line do and why is this required?

Reply:
All white spaces after an exclamation sign in lines starting with exclamation sign are removed. Is required for normalization of excluded paths.

Reply:
@mondeja  Is this normalization really required? In most cases, I've seen people never use spaces after ! . A simple trim() should be good enough. @bishal-pdMSFT  thoughts?

Reply:
Also, can you push a commit to rerun the workflows, I'm unable to see the old logs to check by windows build is failing.

Reply:
I'm not sure if it is really "required". It seems that can be declared with space but the hash will not be the same.

> A simple trim() should be good enough.

I don't follow you. This implementation is not trimming, `! foo` should be converted to `!foo` but `.trim()` does not modificates it (`'! foo'.trim() === '! foo'` is `true`).

Reply:
I understand but what I'm saying is that people don't normally use `! foo`, they'll just use `!foo` directly, as mentioned in our docs.

Reply:
Seems to be some prettier issue for windows. Can you run `npm run format` once to see if it produces any changes?

Reply:
`npm run format` does not produce changes in my Ubuntu, seems only failing in Windows. 

> I understand but what I'm saying is that people don't normally use ! foo, they'll just use !foo directly, as mentioned in our docs.

I haven't used the space in the past and don't have any preference about it. Just implemented here because it is possble to include the space after the exclamation sign using `@actions/glob` API directly:

```javascript
> const glob = require('@actions/glob');
> const globber = await glob.create(['node_modules/.bin/*', '! node_modules/.bin/mkdirp'].join('\n'))
> await globber.glob()
```

But is up to you to include it or not.